srcpkgs/libportal/patches/fix-pytest.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/tests/pyportaltest/__init__.py b/tests/pyportaltest/__init__.py
```

Comment:
Include the patch header:
https://github.com/flatpak/libportal/commit/6cd7c2ab82575b76f876ee2bd2d31f6cb77f022f.patch

Reply:
done